### PR TITLE
Feature/e UI 8379

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8379

### Change description ###
Document uploader in write query views
Refactor to avoid duplication

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by DavidJayakumar — APPROVED** [review]
LGTM

File: src/cases/cases.module.ts [hunk 1]
```diff
@@ -70,6 +69,7 @@ import { CreateCaseEventTriggerResolver } from './resolvers/create-case-event-tr
 // from services
 import * as fromServices from './services';
 import { effects, reducers } from './store';
+import { RpxTranslationModule } from 'rpx-xui-translation';
```
Comment (Daniel-Lam) [inline src/cases/cases.module.ts:72]: @MateiHMCTS - please re-order the imports

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@MateiHMCTS - just one minor change needed; otherwise OK

File: src/cases/cases.module.ts (hunk 1 above)
Comment (MateiHMCTS) [reply src/cases/cases.module.ts:72]: I think it's already ordered as '.' is before 'r'

Comment (Daniel-Lam) [reply src/cases/cases.module.ts:72]: Hmm... interesting. In VSCode, for example, `rxjs` comes before `./users.effects` - can leave for now although the pipeline linter may flag it

Comment (MateiHMCTS) [reply src/cases/cases.module.ts:72]: Nothing from yarn lint

**Review by Daniel-Lam — APPROVED** [review]

**Pull request merged**